src/components/MessageList/MessageList.js

```diff
@@ -148,6 +167,18 @@ const MessageList = ({ messages, handleGoBack }) => {
                       icon="pin"
                       onClick={() => handlePinMessage(msg)}
                     />
+                    {
+                      msg.u._id === authenticatedUserId && (
+                        <ActionButton
```

Comment:
Why do we need an action button? I think it should be messagetoolbox item to keep it semantically aligned.

Reply:
@sidmohanty11 , I have replaced it with the message toolbox item.

Thanks for the suggestion.